app/views/contexts/submission_metadata_context.py

```diff
@@ -11,12 +11,21 @@ def build_submission_metadata_context(
     submitted_on = {
         "term": lazy_gettext("Submitted on:"),
         "descriptions": [
-            {"description": format_datetime(submitted_at, format="dd LLLL yyyy HH:mm")}
+            {
+                "description": lazy_gettext(
+                    "{date} at {time}".format(
+                        date=format_datetime(submitted_at, format="dd LLLL yyyy"),
+                        time=format_datetime(submitted_at, format="HH:mm"),
+                    )
+                )
+            }
         ],
     }
     submission_reference = {
         "term": lazy_gettext("Submission reference:"),
-        "descriptions": [{"description": convert_tx_id(tx_id)}],
+        "descriptions": [
+            {"description": " - ".join(convert_tx_id(tx_id).upper().split("-"))}
```

Comment:
The `convert_tx_id` method should be modified to do this (it is only used for this).

Reply:
`convert_tx_id` now reformatted, thanks!